base/lib/migrate.mustache

```diff
@@ -11,7 +11,7 @@ const r = require('lib/db');
 const table = r.table('_migrations');
 
 const applyMigration = (migrations, then) => {
-  if (migrations.length === 0) return;
+  if (migrations.length === 0) return Promise.resolve();
```

Comment:
Well if it doesn't hit the migrations.length condition it returns a promise, so i figured better to be consistent? But up to you.